mmv1/products/vertexai/FeatureOnlineStoreFeatureview.yaml

```diff
@@ -160,6 +160,10 @@ properties:
               description: |
                 Identifiers of features under the feature group.
               item_type: Api::Type::String
+      - !ruby/object:Api::Type::String
+        name: 'projectNumber'
```

Comment:
Can we use this in a test? https://googlecloudplatform.github.io/magic-modules/develop/test/test/

I checked and current tests don't see it get returned- which I think we expect, since we're only using sources within the same project.

Reply:
yes, I'm not sure on how to add a test for this field as It requires two different GCP project

Reply:
Hi, @rileykarson how do we proceed forward here? Is there a way the magic-modules resource tests can access cross-project resources?

Reply:
Apologies, missed that comment. You can create a new project within a test using the google_project resource: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project